### DISCUSSION TITLE
Probably dumb idea about data storage and transfert

### DISCUSSION THREAD

**Discussion by Yabokaky** [community/community#56958] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi everybody!

My competences are so old and where so low, and the idea so simple that it is probably a stupid / common / old idea... but, just in case it's not so stupid and can be used, here it is...

If my memory is correct, datas are stored and transmitted using a succession of 1 and 0 to code binary values, which are making a text, softawre, movie...
And if all of thoose are differents from each others, their values are allways predeterminated. In example, the value "1" on 32 octets will allways be "0000 0000 0000 0000 0000 0000 0000 0001". Right?

So, why not using a software, related to a database of sheets with a specific number of lines, each line correspondin to a value, and coding a movie by cutting it in sequences, and store them like this :
- value 1 = sheet A ; line 1
-> value 1 = sheet 0000 0001 ; line 0000 0001
- ...
so that we could encode not the information, but the localisation of the information with less octets (here, a total of 16 octets needed to have the emplacement of the 32 octets value), and reduce the volume of the data, decreasing his pollution and time transfert?

Well, that's all. Sorry if it's as stupid as I think it probably is, and sorry for my bad english (but I do have some quality in others sides of life! ;) )

Thanks for reading, and bye!

**Comment by airtower-luna** [comment]
I'm not sure about your specific idea, but encoding data so it takes less space is definitely a thing, see: https://en.wikipedia.org/wiki/Data_compression It's not really my field of expertise so I can't advise on where to learn the mathematical details, but from an engineering point of view compression is an important component in many things. :slightly_smiling_face:
